linkerd/core/src/main/scala/io/buoyant/linkerd/SvcConfig.scala

```diff
@@ -1,13 +1,12 @@
 package io.buoyant.linkerd
 
-import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty, JsonSubTypes}
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty}
 import com.twitter.conversions.time._
 import com.twitter.finagle.{param, Stack}
 import com.twitter.finagle.buoyant.TotalTimeout
 import com.twitter.finagle.service._
 import com.twitter.finagle.buoyant.ParamsMaybeWith
-import com.twitter.util.Duration
-import io.buoyant.config.PolymorphicConfig
+import io.buoyant.namer.BackoffConfig
```

Comment:
I'm not sure if I'm a fan of having this live in the namer package, since I'm pretty sure the backoff config is used in places other than in namers? I think maybe it should be moved someplace where it's visible to the namer module, but not necessarily _into_ that package?

Reply:
Not sure there exists a package that's a perfect fit.  It needs to be visible from both linkerd and namerd and it depends on finagle and config.